q2cli/builtin/tools.py

```diff
@@ -195,7 +195,89 @@ def peek(path):
         click.echo(metadata.format)
 
 
-@tools.command('inspect-metadata',
+@tools.command(name='cast-metadata',
+               short_help='Designate metadata columns as numerical'
+                          ' or categorical.',
+               help='Designating the column types within a metadata file.'
+                    ' Supported column types are numerical or categorical.'
+                    ' Providing multiple file paths to this command will merge'
+                    ' the metadata.',
+               cls=ToolCommand)
+# NOTE: # This gives a potential way to avoid multiple --cast calls but it's
+# fiddely https://stackoverflow.com/a/47730333
+@click.option('--cast', required=True, multiple=True,
+              help='Cast flags for each metadata column that should be'
+              ' specified as either categorical or numeric column types.')
+@click.option('--error-on-extra', is_flag=True,
+              help='Cast flags provided inline that do not exist within the'
```

Comment:
This should somehow indicate that we are talking about metadata _columns_ ("cast flags provided inline that do not correspond to columns names in the provided metadata...").